components/reusable-components/navigation/navigation.module.scss

```diff
@@ -0,0 +1,62 @@
+.navbar {
+  background-color: var(--bg-color);
+}
+
+.navbarBrand {
+  color: #ffff;
+  font-size: 36px;
+  margin-left: 20px;
+  width: 227px;
+  height: 44px;
+  top: 67px;
+  left: 85px;
+}
+
+.navLink1 {
+  color: #ffff;
+  font-size: 25px;
```

Comment:
Please create a variable for the font size in the globals file so it can be used everywhere, and we will have a consistent font size.